src/containers/Storage/StorageGroups/StorageGroups.tsx

```diff
@@ -119,7 +121,19 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes, onShowAll}:
         {
             name: TableColumnsIds.Type,
             header: tableColumnsNames[TableColumnsIds.Type],
-            render: ({value}) => <Label>{value as string || '—'}</Label>,
+            render: ({value, row}) => (
+                <>
+                    <Label>{value as string || '—'}</Label>
+                    {' '}
+                    {row.Encryption && (
+                        <Tooltip content={i18n('encrypted')} placement="right">
```

Comment:
It seems like here we have cursor pointer for elements with tooltips.

So either we need to add such a cursor here, or set default cursor for Pool Name or FDQN (in Nodes), which have tooltips, but are not clickable.

Also the size of the tooltip differs from the size of PoolName tooltip, as well as differs tooltip color in light theme.

Reply:
these are all good points, and there is an incoming PR with these changes :) I don't want to mix refactoring with a new feature which is this PR

cursor: pointer should indicate that the element is clickable, not that is has a tooltip
I believe pool name and fqdn have it because of `Popover` that is used to show tooltips. I'm changing it to `Tooltip` in the next PR, it will also change cursor to default, and equalize the tooltip styles

Reply:
Changed my mind. Will use Popover here instead of Tooltip. They differ in behaviour, e.g. Popover allows its content to be selected, whereas Tooltip hides when cursor leaves the anchor element. I'd rather change other use cases to Tooltip, it looks like a better fit. But the selectability may be an important feature. So it's better for the new case to match the old ones.